src/controller/python/chip/CertificateAuthority.py

```diff
@@ -282,8 +302,13 @@ def NewCertificateAuthority(self, caIndex: Optional[int] = None, maximizeCertCha
             caList[str(caIndex)] = []
             self._persistentStorage.SetReplKey(key='caList', value=caList)
 
+        if (certificateValidityPeriod is None):
```

Comment:
```suggestion
        if certificateValidityPeriod is None:
```

Reply:
Thanks for the suggestion! I have removed the unnecessary parentheses from the if statement for better readability. Marking this as resolved now